changelogs/changelog.yaml

```
@@ -2175,3 +2175,60 @@ releases:
+        operation (https://github.com/ansible-collections/amazon.aws/pull/1611).
+      - backup_vault - fix error when updating tags on a backup vault by using the
+        correct boto3 client methods for tagging and untagging backup resources (https://github.com/ansible-collections/amazon.aws/pull/1610).
+      - ec2_vpc_nat_gateway - adding a boolean parameter called ``default_create``
+        to allow users to have the option to choose whether they want to display an
+        error message or create a NAT gateway when an EIP address is not found. The
+        module (ec2_vpc_nat_gateway) had incorrectly failed silently if EIP didn't
+        exist (https://github.com/ansible-collections/amazon.aws/issues/1295).
+      - ec2_vpc_nat_gateway - fixes to nat gateway so that when the user creates a
+        private NAT gateway, an Elastic IP address should not be allocated. The module
+        had inncorrectly always allocate elastic IP address when creating private
+        nat gateway (https://github.com/ansible-collections/amazon.aws/pull/1632).
+      - module_utils.backup - get_selection_details fix empty list returned when multiple
+        backup selections exist (https://github.com/ansible-collections/amazon.aws/pull/1633).
+      minor_changes:
+      - backup_selection - add validation and documentation for all conditions suboptions
+        (https://github.com/ansible-collections/amazon.aws/pull/1633).
+      - ec2_instance - refactored ARN validation handling (https://github.com/ansible-collections/amazon.aws/pull/1619).
+      - iam_user - refactored ARN validation handling (https://github.com/ansible-collections/amazon.aws/pull/1619).
+      - module_utils.arn - add ``resource_id`` and ``resource_type`` to ``parse_aws_arn``
+        return values (https://github.com/ansible-collections/amazon.aws/pull/1619).
+      - module_utils.arn - added ``validate_aws_arn`` function to handle common pattern
+        matching for ARNs (https://github.com/ansible-collections/amazon.aws/pull/1619).
+      release_summary: This release brings some new modules, features, and several
+        bugfixes.
+    fragments:
+    - 1589-return_false_when_no_change..yml
+    - 1604-c2_vpc_nat_gateway-fails-silently.yml
+    - 1615-no_formatted_with_extra_space.yml
+    - 1632-changes-to-no-allocate-eip-when-connectivity_type=private.yml
+    - 1633-backup-selection-conditions.yml
+    - 1843-iam_instance_profile.yml
+    - 1846-arn-validation.yml
+    - 20230506-autoscaling_group-fix_sanity.yml
+    - 202306012-backup_plan-remove-none-from-nested-params.yml
+    - 20230612-backup_vault-fix-tag-update.yml
+    - 20230627-ci-fixup.yml
+    - release_summary.yml
+    modules:
+    - description: manage IAM instance profiles
+      name: iam_instance_profile
+      namespace: ''
+    - description: gather information on IAM instance profiles
+      name: iam_instance_profile_info
+      namespace: ''
+    release_date: '2023-07-04'
```

```suggestion
    release_date: '2023-07-05'
```

Assuming we'll release Wednesday rather than after the end of your working day today